sefaria/sheets.py

```diff
@@ -1202,3 +1202,19 @@ def change_tag(old_tag, new_tag_or_list):
 	for sheet in SheetSet({"tags": old_tag}):
 		sheet.tags = [tag for tag in sheet.tags if tag != old_tag] + new_tag_list
 		sheet.save()
+
+def get_sheet_categorization_info(does_not_contain):
```

Comment:
I've been puzzling here about what does_not_contain means and what it does. Not clear to me! Either this needs a clarifying comment or a better name or both.